src/home/homework-luks.c

```diff
@@ -63,6 +63,9 @@
                 _x > UINT64_MAX - 4095U ? UINT64_MAX : (_x + 4095U) & ~UINT64_C(4095); \
         })
 
+/* How much larger will the image on disk be than the fs inside it, i.e. the space we pay for the GPT and LUKS2 envelope. */
+#define GPT_LUKS2_OVERHEAD UINT64_C(18874368)
```

Comment:
It's a very specific number, is there a rationale for it? Would be good to specify the source in the comment above

Reply:
this is simply what I measured on current cryptsetup, and it kinda matches docs, i.e. they say 16M JSON area plus some additional binary headers.

Reply:
(will add comment)